README.md

```diff
@@ -72,11 +73,38 @@ Tha last step is to install the service and remove the downloaded files:
 
 ```bash
 /data/dbus-opendtu/install.sh
+/data/dbus-opendtu/restart.sh
 rm main.zip
 ```
 
 Check configuration after that - because the service is already installed and running. In case of wrong connection data (host, username, pwd) you will spam the log-file! Also, check to **set a** proper (minimal) **log level**
 
+### Update the code
+Just grap a copy of the main branch and copy the content to `/data/` e.g. `/data/dbus-opendtu`. The process will preserve your existing config ini. 
+After that call the `install.sh script.
+
+```bash
+wget -O main.zip https://github.com/henne49/dbus-opendtu/archive/refs/heads/main.zip
+unzip main.zip "dbus-opendtu-main/*" -d /data
+rm /data/dbus-opendtu/current.log*
+mv /data/dbus-opendtu-main/config.ini /data/dbus-opendtu-main/config.template
```

Comment:
Sooner or later we should rename our checked-in config.ini to config.init.template. That's simpler.